### PR TITLE
fix(#1122): bulk fixes

### LINKED ISSUE #2033: Import Error - Failed to import transactions (test file included)
**Describe the bug**
When trying to debug more #2032 I came across another issue. I exported a single account in quicken versus every account to narrow down just the bank transfer behavior.

I then attempted to import this. What I discovered is that no transactions were created. Attached is an example QIF file. 

**To Reproduce**
1. Import the attached QIF file

**Expected behavior**
The expected bahavior would be that Bank1 has $250,000 in withdrawals. And that Bank2 has $250,000 in debits. Instead, that is not what we see. No transactions are imported and errors are given. This is different than the issue #2032 where transactions are imported, but transfer direction is incorrect (withdrawal / deposit reversed).

**Your OS:**
- [X] Windows

**MMEX version:**
- [X] development version (git master branch)

### PR DESCRIPTION
**Pull request by vomikan**
#2033
#2032
#2016
#1963
#1700 ??
#1122

make sure your code would work on both windows, linux and osx

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/2040)

### CODE REVIEW

File: src/import_export/qif_import_gui.cpp [hunk 1]
```diff
@@ -357,8 +360,18 @@ bool mmQIFImportDialog::mmReadQIFFile()
     wxLongLong start = wxGetUTCTimeMillis();
     wxLongLong interval = wxGetUTCTimeMillis() - start;
 
-    wxString accName = ""; //TODO: check Account check box
```
Comment (slodki) [inline src/import_export/qif_import_gui.cpp]: :+1:

File: src/import_export/univcsvdialog.cpp [hunk 2]
```diff
@@ -647,10 +668,10 @@ const wxString mmUnivCSVDialog::getCSVFieldName(int index) const
 {
     std::map<int, wxString>::const_iterator it = CSVFieldName_.find(index);
     if (it != CSVFieldName_.end()) {
-        return wxGetTranslation(it->second);
+        return (it->second);
     }
 
-    return _("Unknown");
+    return wxTRANSLATE("Unknown");
```
Comment (slodki) [inline src/import_export/univcsvdialog.cpp]: why?

Comment (vomikan) [reply src/import_export/univcsvdialog.cpp]: it's OK
https://github.com/vomikan/moneymanagerex/blob/%231122/src/import_export/univcsvdialog.cpp#L1154

File: src/import_export/qif_import_gui.cpp (hunk 1 above)
Comment (vomikan) [reply src/import_export/qif_import_gui.cpp]: Bad style to leave todos in the PR but I want to commit as is.

**Pull request merged**